platform/ios/scripts/script_resources/MapboxDemo/MapboxDemo/ViewController.swift

```diff
@@ -0,0 +1,239 @@
+import UIKit
+import Mapbox
+
+/*
+ A simple example that uses the Mapbox Outdoors style to view
+ Yosemite National Park, USA with a polygon visualization of the
+ park boundary that can be interacted with.
+ 
+ A demo access token has been provided for convenience and it may
+ stop working in the future.
+ 
+ You can obtain your own access token from the
+ [Mapbox account page](https://www.mapbox.com/studio/account/tokens/)
+ and add it to this application's Info.plist as the value for
+ MGLMapboxAccessToken
+ */
+class ViewController: UIViewController {
+
+    @IBOutlet weak var mapViewContainer: UIView!
+    @IBOutlet weak var calloutView: UIView!
```

Comment:
Maybe it’s because I’m familiar with our SDK, but naming this `calloutView` made me think this was one of our stock SMCalloutViews. If that’s a common enough impression, maybe we should consider renaming this view to avoid that potential thought-collision.

Reply:
Changed in https://github.com/mapbox/mapbox-gl-native/pull/10524/commits/abfe76a948c41c5592ab43029a583b23fc0497d5